docs/docs/deployment/docker.md

```diff
@@ -39,7 +39,7 @@ OR
 For PostgreSQL database setup with persistent volume:
 ```bash
 curl -LO https://raw.githubusercontent.com/ToolJet/ToolJet/main/deploy/docker/docker-compose-postgres.yaml
```

Comment:
@MithunKinarullathil Could you rename the corresponding file name as well? Currently its `docker-compose-postgres.yml`

Reply:
Done. Although I think it should be the other way around, as in changing everything in README to yml, just not to break anything for existing users?

Reply:
That is alright, as readme was already incorrect.

Reply:
Cool.